src/formats/findFormat.ts

```diff
@@ -0,0 +1,12 @@
+import type { Options } from '../Options';
+import { SuiteEmitter } from '../Suite';
+import * as SpecFormat from './Spec';
+
+type Formatter = (emitter: SuiteEmitter) => void;
+
+export default function findFormat(options: Options): Formatter {
+  switch (options.format) {
+    case 'spec':
+      return SpecFormat.format;
```

Comment:
Typescript seems to understand that we've handled all the possible values of `options.format`, and isn't insisting on a default case here.

My educated guess is that this works because `options.format` is a union type, and can only have specific values.

Reply:
neat!

Reply:
makes sense, nice!